### PR TITLE
adds duration in terms of time.Duration for the frontend

### PR DESCRIPTION
**Pull request by bseto**

### CODE REVIEW

File: backend/game/scribble/handler/gamemaster/gamemaster.go [hunk 1]
```diff
@@ -376,9 +387,7 @@ func (h *Handler) playerSelectTopic() {
 }
 
 type PlayTimeSend struct {
-	LockCanvas bool `json:"lockCanvas,omitempty"`
-	LockChat   bool `json:"lockChat,omitempty"`
-	Score      int  `json:"score,omitempty"`
+	Score int `json:"score,omitempty"`
```
Comment (gordon-cheung) [inline backend/game/scribble/handler/gamemaster/gamemaster.go:390]: What is omitempty?

File: backend/game/scribble/handler/gamemaster/gamemaster.go [hunk 2]
```diff
@@ -278,6 +278,12 @@ type PlayerSelectSend struct {
 	// Choices is a slice of strings that the Chosen client is allowed to choose
 	// from. This Choices field is only sent to the Chosen client
 	Choices []string `json:"choices,omitempty"`
+
+	// Duration is the duration in nanoseconds.
+	Duration time.Duration `json:"duration"`
```
Comment (gordon-cheung) [inline backend/game/scribble/handler/gamemaster/gamemaster.go:283]: What is this duration for?

File: backend/game/scribble/handler/gamemaster/gamemaster.go (hunk 1 above)
Comment (bseto) [reply backend/game/scribble/handler/gamemaster/gamemaster.go:390]: If the field is empty, it'll omit the field in the json

File: backend/game/scribble/handler/gamemaster/gamemaster.go (hunk 2 above)
Comment (bseto) [reply backend/game/scribble/handler/gamemaster/gamemaster.go:283]: hmm i'll update the comment

**Pull request merged**